channels-and-providers/push/pushpad.mdx

```diff
@@ -0,0 +1,48 @@
+---
+title: 'Pushpad'
+description: 'Learn how to use the Pushpad provider to send web push notifications using Novu'
+---
+
+[Pushpad](https://pushpad.xyz) is a web push service that supports sending notifications to all major browsers (Chrome, Firefox, Edge, Safari, etc.) via FCM, Mozilla autopush, Windows Push Notification Services and Apple Push Notification service, with just one simple API.
+
+To configure the Pushpad integration, you will need an active account and you need to have a `Pushpad Auth Token` (from the account settings) and the `Pushpad Project ID` (from the project settings).
```

Comment:
Please can you link to the account settings and the project settings URLs here?

Reply:
@unicodeveloper I have added the requested link for account settings. The link for project settings cannot be added because I would need the project ID to write the URL (but you just need to click "Settings" in the Pushpad project, so it is clear for the user).